installers/helm/README.md

```diff
@@ -0,0 +1,58 @@
+# Crunchy PostgreSQL Operator
+
+This Helm chart uses the `pgo-deployer` container to install the Crunchy PostgreSQL
```

Comment:
This Helm chart installs the Crunchy PostgreSQL Operator by using its “pgo-deployer” container.